examples/run.sh

```diff
@@ -10,6 +10,7 @@ NUM_EPOCHS=1        # number epoches
 MAX_STEPS=-1        # max steps
 GA=1                # gradients accumulation number
 LOG_INTERVAL=1      # log interval
+LOG_LOSS=0          # log loss
```

Comment:
You can directly run ./examples/run.sh --profile --log_loss --spmd_fsdp. The parsing of these parameters is unnecessary.

Reply:
good catch